test/integration-test.py

```diff
@@ -136,17 +136,20 @@ def stat_value(line):
         certStatusWorkBatch = get_stat_line(8014, statline("workbatch", "certificateStatus"))
         certsWorkBatch = get_stat_line(8014, statline("workbatch", "certificates"))
         certsPerNameWorkBatch = get_stat_line(8014, statline("workbatch", "certificatesPerName"))
+        ordersWorkBatch = get_stat_line(8014, statline("workbatch", "orders"))
 
         # sleep for double the configured workSleep for each job
         time.sleep(1)
 
         newCertStatusWorkBatch = get_stat_line(8014, statline("workbatch", "certificateStatus"))
         newCertsWorkBatch = get_stat_line(8014, statline("workbatch", "certificates"))
         newCertsPerNameWorkBatch = get_stat_line(8014, statline("workbatch", "certificatesPerName"))
+        newOrdersWorkBatch = get_stat_line(8014, statline("workbatch", "orders"))
 
         if (certStatusWorkBatch == newCertStatusWorkBatch 
             and certsWorkBatch == newCertsWorkBatch 
-            and certsPerNameWorkBatch == newCertsPerNameWorkBatch):
+            and certsPerNameWorkBatch == newCertsPerNameWorkBatch
+            and ordersWorkBatch == newOrdersWorkBatch):
```

Comment:
this integration test code is a little bit gross and getting out of hand with more tables/jobs being checked. I'd like to refactor it to be a little bit cleaner but ran out of time today. I'll take a pass at that in a follow-up PR unless folks have a strong pref. it be included in this branch.